src/foam/lib/csv/CSVYYMMDDParser.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.lib.csv;
+
+import foam.lib.parse.*;
+import foam.util.SafetyUtil;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+import foam.lib.json.*;
+
+public class CSVYYMMDDParser
```

Comment:
At this point, it isn't really CSV specific, so could you just rename it to YYMMDDParser?